src/main/scala/temple/generate/server/go/GoHTTPStatus.scala

```diff
@@ -2,5 +2,5 @@ package temple.generate.server.go
 
 object GoHTTPStatus extends Enumeration {
   type GoHTTPStatus = Value
-  val StatusBadRequest, StatusUnauthorized, StatusInternalServerError = Value
+  val StatusBadRequest, StatusUnauthorized, StatusNotFound, StatusInternalServerError = Value
```

Comment:
Pedantic, but nothing about this seems to be go specific - could be moved up? But then again it doesn't actually matter

Reply:
Is Go specific because it matches the syntax of the Go `http` package - though if we were supporting different languages I agree we would build a global http enum, that then maps into each languages enum, but ain't nobody got time for dat.